scripts/zones/Mhaura/npcs/Standing_Bear.lua

```diff
@@ -10,7 +10,10 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-    player:startEvent(14)
+    if player:getZPos() > 38.5 then
+        player:startEvent(14)
+    else
+        player:startEvent(235)
```

Comment:
Missing an `end` after this line.

Reply:
i knew i was forgetting something than you :)